src/components/nodes/pages/sup-book/book-page/book-page.tsx

```diff
@@ -198,7 +198,7 @@ const BookPage = async ({node, ...props}: Props) => {
 
             <Link
               href={node.path + "/desk-examination-copy-requests"}
-              className="flex items-start gap-3 text-18 font-normal leading-snug text-stone-dark underline-offset-[5px] hocus:text-archway-dark hocus:decoration-archway-dark hocus:decoration-2"
+              className="flex items-start gap-3 text-[1.8rem] font-normal leading-snug text-stone-dark underline-offset-[5px] hocus:text-archway-dark hocus:decoration-archway-dark hocus:decoration-2"
```

Comment:
@pookmish this is definitely a bandaid fix.

On the otherhand, I did some testing with our summer `twMerge` config just now and we should be able to resolve the `text-18` being dropped with the same custom ``twMerge` config. 

I branched off and pulled over the config just to test in PR #73 👀 Let me know your thoughts and if the twMerge config is something we'd like to merge in!